modules/objdetect/src/qrcode.hpp

```diff
@@ -0,0 +1,56 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2017, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#ifndef QR_CODE_HPP_
+#define QR_CODE_HPP_
+
+using namespace std;
+using namespace cv;
+
+CV_EXPORTS void DetectionQRCode(InputArray in, OutputArray points,
```

Comment:
Rename `DetectionQRCode` to `detectQRCode`. String decode should be passed by reference. What does the function return if there is no QR code in the image?